.github/workflows/pullrequest.yml

```diff
@@ -27,3 +27,35 @@ jobs:
     uses: viamrobotics/rdk/.github/workflows/appimage.yml@main
     secrets:
       GCP_CREDENTIALS: ${{ secrets.GCP_CREDENTIALS }}
+
+  eslint:
+    # Ignore adding a new label not Deploy
+    if: contains(fromJson('["opened", "synchronize", "reopened"]'), github.event.action) || contains(github.event.pull_request.labels.*.name, 'Deploy')
```

Comment:
We don't have a "deploy" label in RDK as far as I know, so I think this is not needed.

Reply:
removed!